### ISSUE TITLE
Feature: broadccast all transactions over Tor

### ISSUE THREAD

**Issue by Kixunil** [Kixunil/btc-rpc-proxy#11]
Having an option to broadcast all transactions over Tor would be a very useful feature since bitcoind doesn't support it directly and some clients don't support it. Having it in the proxy would make sure no transaction leaks over clearnet.

A global `proxied_tx_brodcast` option would be sufficient but perhaps doing it the same way as `fetch_blocks` - per-user AND global is better (consistent). We should also use RPC for testing if the TX would be accepted in the mempool before actually broacasting it.

@dr-bonez how hard do you think it'd be given that some required infrastructure (getting peers) is already implemented?

**Comment by dr-bonez** [comment]
Probably would be pretty trivial. I think we already have a Tor only flag in the config though, so either we should use that, or rename it to something more clear

**Comment by Kixunil** [comment]
Good point. I believe renaming is less confusing because people could incorrectly assume that more operations are torified.
